etc/kayobe/ansible/roles/nova-compute-drain/tasks/instance-info.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Query instances
+  command: >
+    {{ nova_compute_drain_venv }}/bin/openstack
+    server list --host {{ ansible_facts.fqdn }}
```

Comment:
```suggestion
    server list --host {{ ansible_facts.nodename }}
```

Reply:
On the cloud I was testing on their was some funkiness around the compute host names. `nodenmae` is the correct value to use.